src/rulesets/Base/mapreduce.jl

```diff
@@ -51,6 +51,17 @@ end
 ##### `sum(f, x)`
 #####
 
+function rrule(config::RuleConfig{>:HasReverseMode}, ::typeof(sum), f::F, xs::Tuple) where {F}
+    fxs, unmap = rrule(config, map, f, xs)
+    y, unsum = rrule(config, sum, fxs)
```

Comment:
This wants the `sum` rule from https://github.com/JuliaDiff/ChainRules.jl/pull/643/files#diff-9e84213ce96cf861f2b18b2ee71e1dcc4bc9969020802f99705154e13204a1d3R17 

For tuples I think there's no harm in decomposing into `map` then `sum` like this.